front/src/applications/operationalStudies/views/SimulationResults.tsx

```diff
@@ -137,16 +141,16 @@ const SimulationResults = ({
       )}
 
       {/* SIMULATION : SPACE TIME CHART */}
-      <ResizableSection
-        height={manchetteWithSpaceTimeChartHeight}
-        setHeight={setManchetteWithSpaceTimeChartHeight}
-        minHeight={MANCHETTE_WITH_SPACE_TIME_CHART_DEFAULT_HEIGHT}
-      >
-        <div
-          className="simulation-warped-map d-flex flex-row align-items-stretch mb-2"
-          style={{ height: manchetteWithSpaceTimeChartHeight }}
+      {projectionData && projectionData.projectedTrains.length > 0 && (
```

Comment:
Hmm with this condition, when we switch projection, the space taken by the space time disappears resulting with an unpleasant animation where the rest of the simulation goes up